### DISCUSSION TITLE
Document Database Creation

### DISCUSSION THREAD

**Discussion by kassah** [spatie/laravel-multitenancy#402] [Ideas]
Laravel now has an API for creating databases (used in Parallel Testing). I propose we could now add to the documentation an observer for creating databases. This would give users a head start on setting up Laravel Multitenancy without depending on a specific database server.

I have code for this, this discussion is to discuss interest in me creating a PR for this.

**Comment by masterix21** [comment]
Hi @kassah, documentation upgrades are always welcome.
